07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,137 @@
+                    <h6 class="content-title my-3">USER FRIENDLY INTERFACE</h6>
+                    <p class="content-subtitle mb-4 text-secondary fw-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, distinctio!</p>
+                    <a href="#" class="content-link text-secondary text-decoration-none">Read More</a>
+                </div>
+                <div class="col-lg-4 text-center mb-sm-5">
+                    <img src="assets/asteroid.svg" alt="" width="50">
+                    <h6 class="content-title my-3">BUILD THE APP EVERYONE LOVE</h6>
+                    <p class="content-subtitle mb-4 text-secondary fw-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, distinctio!</p>
+                    <a href="#" class="content-link text-secondary text-decoration-none">Read More</a>
+                </div>
+            </div>
+            <div class="row banner">
+                <div class="col-lg-4 text-lg-end text-center">
+                    <img src="./assets/about-img.png" alt="" width="300">
+                </div>
+                <div class="col-lg-8 py-5">
+                    <h1 class="banner-title mb-4 mt-2 fs-sm-4">ESTARTUP <span class="fw-light">UI Design Mobile</span> </h1>
+                    <p class="banner-subtitle mb-4">Consectur adipiscing elit, sed do eiusmood tempor incididunt ut labore dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat</p>
+                    <p class="banner-list text-secondary">> Creative Design</p>
+                    <p class="banner-list text-secondary">> Retina Ready</p>
+                    <p class="banner-list text-secondary">> Easy To Use</p>
+                    <p class="banner-list text-secondary">> Unlimited Features</p>
+                    <p class="banner-list text-secondary">> Unlimited Features</p>
+                </div>
+            </div>
+        </div>
+      </section>
+      <div class="footer">
+        <div class="footer-top bg-black text-white py-5">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-4 col-12 text-center text-lg-start">
+                        <p class="footer-logo mb-1 fw-light">eStartup</p>
+                        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Magni minus et pariatur, saepe illo natus, ex sequi eveniet</p>
+                    </div>
+                    <div class="col-md-3 col-lg-2 col-12 text-center text-lg-start">
+                        <p class="footer-list-head fw-bold mb-3">ABOUT US</p>
+                        <p class="footer-list-item my-1">About Us</p>
+                        <p class="footer-list-item my-1">Features Item</p>
+                        <p class="footer-list-item my-1">Live Streaming</p>
+                        <p class="footer-list-item my-1">Privacy Policy</p>
+                    </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)